src/lib/operators/get_table.cpp

```diff
@@ -23,5 +31,30 @@ std::shared_ptr<AbstractOperator> GetTable::recreate(const std::vector<AllParame
   return std::make_shared<GetTable>(_name);
 }
 
-std::shared_ptr<const Table> GetTable::_on_execute() { return StorageManager::get().get_table(_name); }
+std::shared_ptr<const Table> GetTable::_on_execute() {
+  auto original_table = StorageManager::get().get_table(_name);
+  if (!_excluded_chunks || _excluded_chunks->empty()) {
```

Comment:
For me, having a simple empty vector of excluded chunks makes sense. So why wrap it into an optional?

Reply:
We used it to express the difference between "no chunks are prunable" and "we have not calculated any exclusion lists". the end result is the same though, so i guess an empty vector is the simpler approach.

Reply:
@tbjoern Is this still on the list?